cloudformation/core/eu-west-2/breakglass.yml

```diff
@@ -0,0 +1,290 @@
+                Action:
+                  - "sns:Publish"
+                Resource:
+                  - !ImportValue SlackAlertsTopicArn
+          PolicyName: allow-add-to-group
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Sid: "AllowLambdaServiceToAssumeRole"
+            Effect: "Allow"
+            Action:
+              - "sts:AssumeRole"
+            Principal:
+              Service:
+                - "lambda.amazonaws.com"
+
+  BreakglassRemoveAdminLambdaRole:
+    Type: AWS::IAM::Role
+    Properties:
+      Path: /
+      Policies:
+        - PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "iam:RemoveUserFromGroup"
+                Resource:
+                  - !ImportValue ServiceAdminGroupArn
+              - Effect: Allow
+                Action:
+                  - "sns:Publish"
+                Resource:
+                  - !ImportValue SlackAlertsTopicArn
+          PolicyName: allow-add-to-group
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Sid: "AllowLambdaServiceToAssumeRole"
+            Effect: "Allow"
+            Action:
+              - "sts:AssumeRole"
+            Principal:
+              Service:
+                - "lambda.amazonaws.com"
+
+  BreakglassCleardownLambdaRole:
+    Type: AWS::IAM::Role
+    Properties:
+      Path: /
+      Policies:
+        - PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "iam:RemoveUserFromGroup"
+                  - "iam:GetGroup"
+                Resource:
+                  - !ImportValue ServiceAdminGroupArn
+              - Effect: Allow
+                Action:
+                  - "sns:Publish"
+                Resource:
+                  - !ImportValue SlackAlertsTopicArn
+          PolicyName: allow-add-to-group
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Sid: "AllowLambdaServiceToAssumeRole"
+            Effect: "Allow"
+            Action:
+              - "sts:AssumeRole"
+            Principal:
+              Service:
+                - "lambda.amazonaws.com"
+
+  BreakglassAssumeAdminFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      FunctionName: breakglass-assume-admin
+      Role: !GetAtt BreakglassAssumeAdminLambdaRole.Arn
+      Runtime: nodejs12.x
+      Handler: index.handler
+      Timeout: 5
+      Code:
+        ZipFile: |
+          const AWS = require("aws-sdk");
+          const iam = new AWS.IAM({
+              region: process.env.REGION
+          });
+          const sns = new AWS.SNS({
+            region: process.env.REGION
+          });
+
+          exports.handler = async (event) => {
+              const response = await iam.addUserToGroup({
```

Comment:
Minor: is response meant to be returned or is it just an unused var?

Reply:
Yeah, it's unused, removed it now 👍